subsys/mgmt/Kconfig

```diff
@@ -11,6 +11,7 @@ config MCUMGR_SMP_BT
 	select BT
 	select BT_PERIPHERAL
 	select BT_GATT_DYNAMIC_DB
+	select LEGACY_TIMEOUT_API
```

Comment:
@de-nordic can you please send a patch upstream to mcumgr converting from the legacy timeout API to the new one?

Reply:
yes, will do.

Reply:
Here: https://github.com/apache/mynewt-mcumgr/pull/77

Reply:
Thanks @de-nordic I wasn't planning on waiting for the mcumgr to be fixed for this PR.
So please handle the Kconfig for mcumgr once your PR get's in.

Reply:
OK, Thx

Reply:
mcumgr library was upgraded, lets's remove the line then

Reply:
@nvlsianpu Thanks. Was the zephyr manifest updated as well?

Reply:
@joerchan yes

Reply:
Removed now.